lib/components/editable_select_component.dart

```diff
@@ -113,6 +113,7 @@ class EditableSelectComponent extends SelectComponent {
   }
 
   get current_input_value => findPart("display_input").value;
+  get name => findPart("input").name;
 
```

Comment:
SelectComponent doesn't need this getter because it loads value for "name" attribute from DOM properly.